thoth/workflow_helpers/utils.py

```diff
@@ -15,9 +15,20 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-"""This is the main script of the template project."""
+"""These are common scripts that can be reused."""
 
-from template.version import __version__
```

Comment:
this should not be removed, we need __version__ in a file called `version.py` for Kebechet, https://github.com/thoth-station/kebechet/tree/master/kebechet/managers/version#kebechet-version-manager